docs/modules/servers/pages/distributed/configure/cassandra.adoc

```diff
@@ -191,4 +191,11 @@ If enabled, regular consistency level is used for read transactions for message.
 in stale reads as the system.paxos table will not be checked for latest updates. Better performance are expected
 by turning it off. Note that reads performed as part of write transactions are always performed with a strong
 consistency.
+
+| message.write.strong.consistency.unsafe
+| Optional. Boolean, defaults to true. Disabling should be considered experimental and unsafe.
+If enabled, Lightweight transactions will no longer be used upon messages operation (table `imapUidTable`).
```

Comment:
Shouldn't it be if disabled? (same for below docs)

Reply:
No I was thinking the way you turn it, should be disabled for `message.write.strong.consistency.unsafe`, not for the other parameters. Let me extend it.

Here if disabled makes sense, because then you say LWT are not used in this case. However, for `message.read.strong.consistency` and `mailbox.read.strong.consistency` it should stay if enabled, as you say regular consistency level (so LWT) is used.

When I said same for below docs, that was for the other doc where you mention `message.write.strong.consistency.unsafe`